parts/29-server.sh

```diff
@@ -38,6 +38,38 @@ server_users() { ## : Count of different classifications of users
 	EOF
 }
 
+server_oidc() { ## : How many users logged in with OIDC
+	meta <<-EOF
+		ADDED: 21
+	EOF
+	handle_help "$@" <<-EOF
+	EOF
+
+	op="="
+	if (( $# > 1 )); then
+		op="$2"
+	fi
```

Comment:
```suggestion
	op="="
	if [[ -z "$arg_op" ]]; then
		op="$arg_op"
	fi
```

There's a new "default value" syntax that can be used but...accepting strings like `=` is going to completely break that parsing so, we just leave this the old way.